src/pages/Admin/EntranceExperience/EntranceExperience.tsx

```diff
@@ -128,13 +133,13 @@ const EntranceExperience: React.FC<EntranceExperienceProps> = ({
               register={register}
               fieldName="entrance"
               showTitle={false}
+              editing={venue.entrance}
+              errors={errors.entrance}
             />
           </S.ItemBody>
         </S.ItemWrapper>
 
-        <Button type="submit" disabled={!dirty}>
-          Save
-        </Button>
+        <Button type="submit">Save</Button>
```

Comment:
Why did we remove the `disabled` from this?

Reply:
The fields have validation already. And the `dirty` state is not changing to true, after calling `clear` or `remove` from `useForm`

Reply:
Makes sense.

I'm not sure that our other forms have the 'disable the submit button' sort of logic on them anyway do they? If there is a pattern that we already have that handles this well that you can copy from, that would be cool. But if this was a one off/different from our standards, then no need to try and implement it.

Reply:
Yeah, this disable pattern is new and should be removed imo, just to keep it consistent with the other ux flow.

Reply:
Sounds good to me